UsingTheServer.md

```diff
@@ -0,0 +1,157 @@
+# Using Your Own Application
+
+The server is currently set up to run with a default cobspec application.  This guide will help you set up the server for use with your own application.
+
+## Adding a module
+
+Much like the cobspec application, your application should be added as a module to the project structure.
+
+server    
+|_ cobspec      
+|_ core    
+|_ YOUR_APPLICATION
+
+Set up your module with Gradle using GroupID `com.saralein` and ArtifactID `YOUR_APPLICATION`, where `YOUR_APPLICATION` is whatever name you choose.
+
+Your module should contain a build.gradle.  settings.gradle in the project root should be modified to include your application:
+
+`rootProject.name = 'server'`    
+ `include 'core', 'cobspec', 'YOUR_APPLICATION'`    
+
+## Updating your application build.gradle
+
+The build.gradle in your application module should be updated to include the following:
+
+```
+group 'com.saralein'
+version '1.0'
+   
+apply plugin: 'com.github.johnrengelman.shadow'
+apply plugin: 'java'
+apply plugin: 'application'
+   
+mainClassName = 'com.saralein.YOUR_APPLICATION.Main'
+sourceCompatibility = 1.8
+   
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.github.jengelman.gradle.plugins:shadow:2.0.1'
```

Comment:
What does this do?

Reply:
I kept reading people having to use plugins to make fat jars in Gradle, so Shadow is a plugin for that.  However, I found another way which doesn't depend on third-party stuff, so I've updated the `build.gradle` and `md`.